gui/src/common/regex-helpers.ts

```diff
@@ -55,7 +55,7 @@ export const udxBucketNameRegex = helpers.regex(
   /^[a-z0-9-]{0,51}[a-z0-9]$/
 );
 // tslint:disable-next-line
-export const udxBucketNameTooltipMessage = `To identify the S3 bucket correctly, the Bucket name will always use "udx" as its prefix. The bucket name must be of minimum 5 characters and maximum 56 characters.
+export const udxBucketNameTooltipMessage = `To identify the S3 bucket correctly, the Bucket name will always use "ldp" as its prefix. The bucket name must be of minimum 5 characters and maximum 56 characters.
```

Comment:
I think we dont require this now as per our last conversation

Reply:
Removed the constant